project/project_test.go

```diff
@@ -201,7 +201,7 @@ func TestParseWithMultipleComposeFiles(t *testing.T) {
 	configThree := []byte(`
```

Comment:
Idea would be to add a `configFour` with the `mem_limit: 40000000` to make sure it still works.